ckeditor/static/ckeditor/ckeditor-init.js

```diff
@@ -7,6 +7,12 @@
   }
 
   var $ = jQuery || djangoJQuery;
+
+  var el = $('.django-ckeditor-widget');
```

Comment:
At this point, the div with the class `django-ckeditor-widget` has not yet been loaded, so `el.length` will be zero. Perhaps this code should be moved into the jQuery ready function.